### DISCUSSION TITLE
Is type-graphql production trustworthy?

### DISCUSSION THREAD

**Discussion by kicks321** [MichalLytek/type-graphql#1377] [General]
We are looking to migrate our entire Graphql Stack over to type-graphql due to the feasibility behind writing resolvers.  

Right now we use our own custom built ES6 classed based approach (similar to how controllers are built in REST).  

Is type-graphql used in production by other clients, how long are we looking at solid support, and is there a multitude of contributions aside from the primary owner of this library?

**Comment by MichalLytek** [comment]
> is there a multitude of contributions aside from the primary owner of this library?

I am a sole maintainer, but I see there's a bigger and bigger community around TypeGraphQL which propose new features and even open PRs with fixes.

TypeGraphQL is now a mature framework, that's why there's not too many commits recently as there's even no big bugs to fix.
V2.0 with GraphQL v16 support is incoming, then I plan to rewrite the internals to make it more modular, maintainable and extendable.

**Comment by guguji5** [comment]
hi @kicks321 , have you met the issue of Inheritance ?
https://github.com/MichalLytek/type-graphql/discussions/1407

**Comment by thiago-pinheiro** [comment]
Hello .. Good night. Yes, I already worked well with react inheritance and
also next at mobiauto where I interned


Em qui., 5 de jan. de 2023 às 00:12, Damon ***@***.***>
escreveu:
